EasyFinance.Persistence/Mapping/AccessControl/UserProjectConfiguration.cs

```diff
@@ -15,6 +15,12 @@ public override void ConfigureEntity(EntityTypeBuilder<UserProject> builder)
             builder.HasOne(p => p.User)
                 .WithMany()
                 .IsRequired();
```

Comment:
```suggestion
                .WithMany();
```

I suggest you to remove this `isRequired` tag from user thinking that now the user can be null.